Chummer/Utilities/frmDiceRoller.cs

```diff
@@ -101,22 +103,40 @@ private void cmdRollDice_Click(object sender, EventArgs e)
                     if (chkCinematicGameplay.Checked)
                         intTarget = 4;
 
-                    if (intResult >= intTarget)
+                    if (intResult >= intTarget) {
```

Comment:
Try to pay attention to the curly bracket standards already established in the rest of the code's style.